src/main/webapp/WEB-INF/jsp/user-page.jsp

```diff
@@ -46,13 +46,16 @@
         </form>
       </div>
       <div id="naviposts">
-        <a href="">Posts</a> <a href="">Gallery</a>
+        <button id="posts-button" onclick="switchTab('posts')">Posts</button> <button id="gallery-button" onclick="switchTab('gallery')">Gallery</button>
```

Comment:
Maybe add a description about what these mean? (So people know how to try it out next week.)